### ISSUE TITLE
Commit: ed96e483 - also can\t build :( more missing files needed:)

### ISSUE THREAD

**Issue by VictorVG** [processhacker2/processhacker#187]
```
$ build_release.cmd
Branch: master
Commit: ed96e483
Version: 3.0.1063

[2017-11-03] dmex Add missing file from previous commit                             (ed96e483)
[2017-11-03] dmex Update plugin window layout                                       (203c557c)
[2017-11-03] dmex Add deprecated ExtraPlugins plugin to blocklist                   (13d553a3)
[2017-11-03] dmex Update plugin manager window                                      (0297b865)
[2017-11-03] dmex OnlineChecks: Use http wrapper                                    (60a10900)

Building ProcessHacker (x32)...
[ERROR] sysscmem.c(342): error C2065: IDC_COMMIT: необъявленный идентификатор [C:\dev\ProcessHacker\check\ProcessHacker\trunk\ProcessHacker\ProcessHacker.vcxproj]
plugin.c(215): error C2220: предупреждение обработано как ошибка - файл "object" не создан [C:\dev\ProcessHacker\check\ProcessHacker\trunk\ProcessHacker\ProcessHacker.vcxproj]
plugin.c(215): warning C4090: функция: разные квалификаторы "const" [C:\dev\ProcessHacker\check\ProcessHacker\trunk\ProcessHacker\ProcessHacker.vcxproj]
Для продолжения нажмите любую клавишу . . .
```

**Comment by VictorVG** [comment]
After I added `#define IDC_COMMIT  134` in to ./ProcessHacker/resource.h then `[ERROR] sysscmem.c(342): error C2065: IDC_COMMIT: unknown identifies` is fixed, but exist's error C2220 in to ProcessHacker/plugin.c::215:

```
    static PCWSTR PhpPluginBlocklist[] =
    {
        L"CommonUtil.dll",
        L"ExtraPlugins.dll"
    };
    BOOLEAN blacklistedPlugin = FALSE;
    PH_STRINGREF baseName;
    PPH_STRING fileName;

    baseName.Buffer = Information->FileName;
    baseName.Length = Information->FileNameLength;

    for (ULONG i = 0; i < ARRAYSIZE(PhpPluginBlocklist); i++)
    {
        if (PhEndsWithStringRef2(&baseName, PhpPluginBlocklist[i], TRUE))
        {
            blacklistedPlugin = TRUE;
            break;
        }
    }
```
I don't know as fix it. :(

**Comment by dmex** [comment]
@VictorVG 

In the future just use the Github tracker and nothing else... Sending me an email, private message, forum topics and forwarding other messages ends up sending 6-7 emails and multiple notifications for the exact same thing.

**Comment by VictorVG** [comment]
Ok! But problem in to ProcessHacker/plugis.c:215 is exist - r1066 - build failure.

$ build_release.cmd
Branch: master
Commit: 044a1f30
Version: 3.0.1066

[2017-11-03] dmex Add file from previous commit                                     (044a1f30)
[2017-11-03] dmex Move PhGetShieldBitmap, Fix bitmap flags                          (02812861)
[2017-11-03] dmex Remove unused resources                                           (fa21e513)
[2017-11-03] dmex Add missing file from previous commit                             (ed96e483)
[2017-11-03] dmex Update plugin window layout                                       (203c557c)

Building ProcessHacker (x32)...
[ERROR] plugin.c(215): error C2220: предупреждение обработано как ошибка - файл "object" не создан [c:\dev\ProcessHacker\check\ProcessHacker\trunk\ProcessHacker\ProcessHacker.vcxproj]
plugin.c(215): warning C4090: функция: разные квалификаторы "const" [c:\dev\ProcessHacker\check\ProcessHacker\trunk\ProcessHacker\ProcessHacker.vcxproj]
